ApplicationLibCode/Application/RiaPreferences.cpp

```diff
@@ -270,9 +274,13 @@ RiaPreferences::RiaPreferences()
 
     CAF_PDM_InitFieldNoDefault( &m_osduPreferences, "osduPreferences", "osduPreferences" );
     m_osduPreferences = new RiaPreferencesOsdu;
+    CAF_PDM_InitField( &m_deleteOsduToken, "deleteOsduToken", false, "" );
+    caf::PdmUiPushButtonEditor::configureEditorLabelHidden( &m_deleteOsduToken );
```

Comment:
Disable IO to avoid storing bool value:  `m_deleteOsduToken.xmlCapability()->disableIO();`